### PR TITLE
Fix some small sfinv issues

### PR DESCRIPTION
**Pull request by rubenwardy**

### CODE REVIEW

File: mods/sfinv/api.lua [hunk 1]
```diff
@@ -29,7 +29,7 @@ end
 function sfinv.get_nav_fs(player, context, nav, current_idx)
 	-- Only show tabs if there is more than one page
 	if #nav > 1 then
-		return "tabheader[0,0;tabs;" .. table.concat(nav, ",") .. ";" .. current_idx .. ";true;false]"
+		return "tabheader[0,0;sfinv_nav_tabs;" .. table.concat(nav, ",") .. ";" .. current_idx .. ";true;false]"
```
Comment (SmallJoker) [inline mods/sfinv/api.lua]: Line too long.

File: mods/sfinv/api.lua [hunk 2]
```diff
@@ -84,10 +84,16 @@ function sfinv.get_formspec(player, context)
...
 		local old_page = context.page
-		context.page = sfinv.get_homepage_name(player)
-		assert(sfinv.pages[context.page], "[sfinv] Invalid homepage")
-		minetest.log("warning", "[sfinv] Couldn't find " .. dump(old_page) .. " so using switching to homepage")
-		return sfinv.get_formspec(player, context)
+		local home_page = sfinv.get_homepage_name(player)
+		if context.page == home_page then
+			minetest.log("error", "[sfinv] Couldn't find " .. dump(old_page) .. ", which is also the old page")
+			return ""
+		else
```
Comment (SmallJoker) [inline mods/sfinv/api.lua]: Redundant indents and `else` case after `return ""`.

File: mods/sfinv/api.lua [hunk 3]
```diff
@@ -84,10 +84,16 @@ function sfinv.get_formspec(player, context)
...
+			context.page = home_page
+			assert(sfinv.pages[context.page], "[sfinv] Invalid homepage")
+			minetest.log("warning", "[sfinv] Couldn't find " .. dump(old_page) .. " so using switching to homepage")
```
Comment (SmallJoker) [inline mods/sfinv/api.lua]: Line too long.

File: mods/sfinv/api.lua [hunk 4]
```diff
@@ -84,10 +84,16 @@ function sfinv.get_formspec(player, context)
 		return page:get(player, context)
 	else
 		local old_page = context.page
-		context.page = sfinv.get_homepage_name(player)
-		assert(sfinv.pages[context.page], "[sfinv] Invalid homepage")
-		minetest.log("warning", "[sfinv] Couldn't find " .. dump(old_page) .. " so using switching to homepage")
-		return sfinv.get_formspec(player, context)
+		local home_page = sfinv.get_homepage_name(player)
+		if context.page == home_page then
+			minetest.log("error", "[sfinv] Couldn't find " .. dump(old_page) .. ", which is also the old page")
```
Comment (SmallJoker) [inline mods/sfinv/api.lua]: Line too long.

**Comment by paramat** [comment]
I'm ok with this (through trust) once those line comments are attended to.

**Comment by rubenwardy** [comment]
Done!

**Comment by paramat** [comment]
db08c0d721e056862adf4af4ac19bd800c747f6a
a4b94a334754e5e7a5343e5d3caafdd76b94daff

**Pull request closed without merging**